### DISCUSSION TITLE
Private profile options

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32528] [Other Feature Feedback, Questions, & Ideas]
Rather than making a user profile completely private, more customisable options would be better:

- Private `Starred Repos`
- Private `Activity`
- Private `Packages`
- Private `Projects`

**Comment by zachMitchell** [comment]
Absolutely agree here. Being able to display these settings individually would provide some flexibility in regards to privacy. Having contributions show up, while also hiding something like followers or activity would be fantastic.

**Comment by vojkny** [comment]
Related issue: https://github.com/orgs/community/discussions/31372 - please have a look here 😢

**Reply by bebo13133** [reply]
:(
